server/models/article.js

```diff
@@ -5,8 +5,12 @@ export default (sequelize, DataTypes) => {
       allowNull: false,
       unique: true
     },
+    categoryId: {
+      type: DataTypes.INTEGER,
+      allowNull: false
+    },
     title: {
-      type: DataTypes.STRING,
+      type: DataTypes.INTEGER,
       allowNull: false
     },
     body: {
```

Comment:
@Veraclins please, can you change the body datatype to text instead of string for the article models. Thank you

Reply:
Ok. I will do that.